source/site-administrators/engine/engine-site-security-guide.rst

```diff
@@ -442,3 +443,45 @@ expression evaluates to false, access is denied. The following expressions can b
 *   ``hasAnyRole({'role1', 'role2'})``
 *   ``permitAll()``
 *   ``denyAll()``
+
+-------------------------
```

Comment:
Move this into the single sign-on section. Put this as the first second and call it "SAML 2 Single Sign-On without Crafter Profile" and put it above the SSO with Profile section.